projects/cn0506_mii/zc706/system_constr.xdc

```diff
@@ -0,0 +1,60 @@
+set_property -dict {PACKAGE_PIN AB27  IOSTANDARD LVCMOS25} [get_ports mii_rx_er_b]                 ; ## D20 FMC_LPC_LA17_CC_P
+set_property -dict {PACKAGE_PIN AG30  IOSTANDARD LVCMOS25} [get_ports mii_rx_dv_b]                 ; ## H29 FMC_LPC_LA24_N
+set_property -dict {PACKAGE_PIN AH26  IOSTANDARD LVCMOS25} [get_ports {mii_rxd_b[0]}]              ; ## H22 FMC_LPC_LA19_P
+set_property -dict {PACKAGE_PIN AH27  IOSTANDARD LVCMOS25} [get_ports {mii_rxd_b[1]}]              ; ## H23 FMC_LPC_LA19_N
+set_property -dict {PACKAGE_PIN AG26  IOSTANDARD LVCMOS25} [get_ports {mii_rxd_b[2]}]              ; ## G21 FMC_LPC_LA20_P
+set_property -dict {PACKAGE_PIN AG27  IOSTANDARD LVCMOS25} [get_ports {mii_rxd_b[3]}]              ; ## G22 FMC_LPC_LA20_N
+set_property -dict {PACKAGE_PIN AG29  IOSTANDARD LVCMOS25} [get_ports mii_tx_clk_b]                ; ## G28 FMC_LPC_LA25_N
+set_property -dict {PACKAGE_PIN AF30  IOSTANDARD LVCMOS25 SLEW FAST} [get_ports mii_tx_en_b]       ; ## H28 FMC_LPC_LA24_P
+set_property -dict {PACKAGE_PIN AH28  IOSTANDARD LVCMOS25 SLEW FAST} [get_ports {mii_txd_b[0]}]    ; ## H25 FMC_LPC_LA21_P
+set_property -dict {PACKAGE_PIN AH29  IOSTANDARD LVCMOS25 SLEW FAST} [get_ports {mii_txd_b[1]}]    ; ## H26 FMC_LPC_LA21_N
+set_property -dict {PACKAGE_PIN AK27  IOSTANDARD LVCMOS25 SLEW FAST} [get_ports {mii_txd_b[2]}]    ; ## G24 FMC_LPC_LA22_P
+set_property -dict {PACKAGE_PIN AK28  IOSTANDARD LVCMOS25 SLEW FAST} [get_ports {mii_txd_b[3]}]    ; ## G25 FMC_LPC_LA22_N
+
+set_property -dict {PACKAGE_PIN AD25  IOSTANDARD LVCMOS25 PULLUP true} [get_ports mdio_fmc_b]      ; ## H31 FMC_LPC_LA28_P
+set_property -dict {PACKAGE_PIN AE26  IOSTANDARD LVCMOS25} [get_ports mdc_fmc_b]                   ; ## H32 FMC_LPC_LA28_N
+
+set_property -dict {PACKAGE_PIN AB14  IOSTANDARD LVCMOS25} [get_ports reset_b]                     ; ## H20 FMC_LPC_LA15_N
+set_property -dict {PACKAGE_PIN AF29  IOSTANDARD LVCMOS25} [get_ports link_st_b]                   ; ## G27 FMC_LPC_LA25_P
+set_property -dict {PACKAGE_PIN AK26  IOSTANDARD LVCMOS25} [get_ports mii_crs_b]                   ; ## D24 FMC_LPC_LA23_N
+set_property -dict {PACKAGE_PIN AJ26  IOSTANDARD LVCMOS25} [get_ports led_0_b]                     ; ## D23 FMC_LPC_LA23_P
+set_property -dict {PACKAGE_PIN AJ30  IOSTANDARD LVCMOS25} [get_ports led_bl_c_c2m]                ; ## D26 FMC_LPC_LA26_P
+set_property -dict {PACKAGE_PIN AK30  IOSTANDARD LVCMOS25} [get_ports led_bl_a_c2m]                ; ## D27 FMC_LPC_LA26_N
+set_property -dict {PACKAGE_PIN AE18  IOSTANDARD LVCMOS25} [get_ports led_br_c_c2m]                ; ## G18 FMC_LPC_LA16_P
+set_property -dict {PACKAGE_PIN AE17  IOSTANDARD LVCMOS25} [get_ports led_br_a_c2m]                ; ## G19 FMC_LPC_LA16_N
+
+create_clock -name rx_clk_1       -period  40.0 [get_ports mii_rx_clk_a]
+create_clock -name rx_clk_2       -period  40.0 [get_ports mii_rx_clk_b]
+create_clock -name ref_clk_125    -period  8.0 [get_ports ref_clk_125_p]
```

Comment:
This port is not part of the design. Should be removed

Reply:
Done.